doc/source/cli_topo.md

```diff
@@ -0,0 +1,265 @@
+(cli-topo)=
+
+# Topo workflow
+
+## Summary
+
+The topo workflow is designed to provide users with comprehensive information about their elevation model,
+including terrain attributes (i.e. slope, hillshade, aspect, etc.) and statistical analyses (i.e. mean, max, min, etc.)
+
+
+
+:::{figure} imgs/topo_workflow_pipeline.png
+:width: 100%
+:::
+
+## Available command line
+
+Run the workflow :
+
+```{code}
+xdem topo --config config_file.yaml
+```
+
+To display a template of all available configuration options for the YAML file, use the following command:
+
+```{code}
+xdem topo --display-template-config
+```
+
+## Detailed description of input parameters
+
+```{eval-rst}
+.. tabs::
+
+   .. tab:: inputs
+
+     **Required:** Yes
+
+     Elevation input information.
+
+     .. csv-table:: Inputs parameters for elevation
+        :header: "Name", "Description", "Type", "Default value", "Required"
+        :widths: 20, 40, 20, 10, 10
+
+        "path_to_elev", "Path to reference elevation", "str", "", "Yes"
+        "force_source_nodata", "No data elevation", "int", "", "No"
+        "path_to_mask", "Path to mask associated to the elevation", "str", "", "No"
+        "from_vcrs", "Original vcrs", "int, str", "EGM96", "No"
+        "to_vcrs", "Destination vcrs", "int, str", "EGM96", "No"
+
+     .. note:: For setting the vcrs please refer to :doc:`vertical_ref`
+
+     .. code-block:: yaml
+
+         inputs:
+           reference_elev:
+             path_to_elev: "path_to/ref_dem.tif"
+             force_source_nodata: -32768
+             from_vcrs: "EGM96"
+             to_vcrs: "Ellipsoid"
+
+   .. tab:: statistics
+
+      **Required:** No
+
+      Statistics step information. This section relates to the computed statistics:
+        1. If no block is specified, all available statistics are calculated by default.
+        [mean, median, max, min, sum, sum of squares, 90th percentile, LE90, nmad, rmse, std, valid count, total count,
+        percentage valid points, inter quartile range]
+        2. If a block is specified but no statistics are provided, then no statistics will be computed.
```

Comment:
You might need to add a blank line, as the linebreak does not show in the final doc.